src/styles/theme.js

```diff
@@ -9,6 +9,12 @@ const theme = {
         keys: breakpointsKeys,
         values: breakpointsValues,
     }),
+    zIndex: {
+        drawer: 10000,
+        modal: 11000,
+        prompt: 12000,
+        banner: 13000,
+    },
```

Comment:
These constants look different than the `variables.scss` constants.

Reply:
Correct

Reply:
The only difference is prompt and banner. Really, I don't think we should have prompt in there, but for now it is and should be higher than drawer and modal, but lower than banner.